### DISCUSSION TITLE
how to send message to vscode client to update the content in the editor

### DISCUSSION THREAD

**Discussion by sujatha2206** [microsoft/vscode-discussions#1189] [Extension Development QnA]
I am working on the debug adapter extension .I want  to implement a feature that changes debugger program.I am using source reference to stream content from backend and initial content is populated successfully.
I using the loadedsource event with the changed flag ,But it's not updated the content.

**Comment by isidorn** [comment]
Do you still hit this issue? That even should be able to update the content of the in-memory editor.
If you still see this, then please open a new issue in VS Code repository. Thank you 🙏
